modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
+
+.Procedure
+To customize and configure the cluster roles of all the projects:
+
+. In the *Developer* perspective, navigate to the *Search* view.
```

Comment:
Cluster admins work maybe more time in the admin perspective, we have a search there as well which works similar. Maybe we can just drop the perspective recommendation?

```suggestion
. Navigate to the *Search* view.
```

Reply:
Sounds fair, I can rephrase without using "**Developer** perspective".